### PR TITLE
Feature/9 update nhinx

### LINKED ISSUE #9: Update Nginx-Ingress and outsource static content into separate web server

### PR DESCRIPTION
**Pull request by jsprey**
Resolves #9

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- [#9] Updated the nginx ingress controller to version 1.3.0. More information can be found [ere](https://github.com/kubernetes/ingress-nginx/releases/tag/controller-v1.3.0)
```
Comment (nhinze23) [inline CHANGELOG.md]: here

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
...
+
+### Removed
+- [#9] Static content such as the ces-theme is no longer served by the ingress controller. The static content is generally outsources to another webserver called [k8s-static-webserver](https://github.com/cloudogu/k8s-static-webserver)
```
Comment (nhinze23) [inline CHANGELOG.md]: outsourced

Comment (nhinze23) [inline CHANGELOG.md]: https://github.com/cloudogu/nginx-static

File: dogu.json [hunk 3]
```diff
@@ -15,12 +15,13 @@
...
     {
-      "type": "dogu",
-      "name": "registrator"
+      "type": "client",
+      "name": "k8s-dogu-operator"
     },
     {
       "type": "client",
-      "name": "k8s-dogu-operator"
+      "name": "cesapp",
```
Comment (nhinze23) [inline dogu.json:23]: Nice

File: CHANGELOG.md (hunk 1 above)
Comment (jsprey) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (jsprey) [reply CHANGELOG.md]: Done

Comment (jsprey) [reply CHANGELOG.md]: Hawkeye :eye_speech_bubble:

File: dogu.json (hunk 3 above)
Comment (jsprey) [reply dogu.json:23]: I improved it further with version < 0.0.0

Comment (jsprey) [reply dogu.json:23]: Done

**Pull request merged**